extensions/extensions-helm/aissemble-kafka-chart/values.yaml

```diff
@@ -19,9 +14,15 @@ kafka:
   interBrokerListenerName: CLIENT
   # Listener Ports Configuration
   listeners:
-    - "CLIENT://0.0.0.0:9093"
```

Comment:
S: We will want to provide a migration for these parameters in case they've been overridden in the downstream chart.